pkgs/top-level/all-packages.nix

```diff
@@ -2062,7 +2062,7 @@ in
 
   horst = callPackage ../tools/networking/horst { };
 
-  host = callPackage ../tools/networking/host { };
+  host = bind.host;
```

Comment:
My user expectation would be at least, that I get `host`, when pulling `dnsutils` into systemPackages. How about an additional symlink?